test/pre-post-extension.js

```diff
@@ -0,0 +1,56 @@
+import chai, {expect} from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import Stipulate from '../src';
+
+chai.use(chaiAsPromised);
+
+class ExtStip extends Stipulate {
```

Comment:
If you adapt to what I said about hooks, this entire file can be deleted. Especially since these tests are actually testing class inheritance more so than actual functionality.